index.html

```diff
@@ -0,0 +1,136 @@
+          <span class="bar"></span>
+      </div>
+  </nav>
+  <div class="site-content">
+    <div class="wanted">
+      <ul>
+        <li class="jof">f</li>
+      </ul>
+        <ul class="wina">
+        <li>English</li>
+        <li>Mypage</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <nav class="navbar">
+      <div>
+      <img class="lolo" src="Assets/CC_Global_Summit_2015 logo.png" alt="CC_Global_Summit_2015" />
+    </div>
+    <div>
+      <ul class="menu">
+        <li><a id="a" href="/About .html">About</a></li>
+        <li class="item"><a href="#program">Program</a></li>
+        <li class="item"><a href="#jopn">Join</a></li>
+        <li class="item"><a href="#sponser">Sponser</a></li>
+        <li class="item"><a href="#news">News</a></li></li>
+        <li class="itoa">CC Campaign</li>
+      </ul>
+    </div>
+    </nav>
+</header>
+  <body>
+           <div class="contents">
+                    <h2 class="contents-head">"Hello! Sharing World"</h2>
+                    <h1 class="contain">CREATIVE COMMONS<BR>GLOBAL SUMMIT<BR>2015</h1>
+                      <div class="load">
+                        <P class="paragraph">
+                            A joyful celebration in the value of openness and  sharing, ceating <br> a positive change with people from over 80 countries is taking place, in october Korea
+                        </P>
+                        <h3 class="date">2015.10.15(THU) ~ 16(FRI)</h3>
+                        <p >@ National Museam of Korea, Art Center Nabi and more</p>
+                      </div>
+                </div>
+            </div>
+            <section class="card-section">
+              <div class="card-container">
+                  <h1 class="box">Main Programs</h1>
+                  <div id="woth">
+                    <div class="power">
+                      <div class="card-one">
+                        <img class="icon1" src="Assets/main-icon/Screenshot (207).png" alt="counter">
+                        <div class="info">
+                        <h1 class="card-info" >lecture</h1>
+                      </div>
+                      <p>
+                        Listen to the speakers from various countries about the messeges of sharing and opening.
+                      </p>
+                      </div>
+                <div class="card-one">
+                    <img class="icon1" src="Assets/main-icon/Screenshot (208).png" alt="counter" />
+                    <div class="info" >
+                    <h1 class="card-info">CC Exhibition</h1>
+                  </div>
+                  <p>
+                    Appreciate various creations applying CC license of artists, from Art center Nabi
+                  </p>
+                </div>
+                </div>
+                <div class="card-one">
+                  <img class="icon1" src="Assets/main-icon/Screenshot (209).png" alt="counter" />
+                  <div class="info">
```

Comment:
- [ ] I would kindly suggest you stick to the original design as much as possible, currently, the `main program` page of your design should have a black background including the cards and also there is a small red line under the `main program` text but yours is missing that currently
- [ ] Also the `SEE MORE PROGRAM` is missing kindly add it

| Behance Design | Your Design |
| ------------- | ------------- |
| ![cin-main](https://github.com/maserekafelex/Final-project/assets/78634841/3b823419-5c54-413c-b703-e2c6050a18bd)| ![mak-main](https://github.com/maserekafelex/Final-project/assets/78634841/a10679ee-b8b2-4798-a9a2-b114372b4d9b) |